### PR TITLE
Privacy and testing

### PR DESCRIPTION
**Pull request by hopesgit**
## Overview:

Fixes #

* [x] All tests passing
* [x] Lint code
* [x] Checked local host
* [ ] Updated README
There is one partially-implemented test in the welcome spec testing oauth implementation that is currently xit'd out. Needs OmniAuth-based testing.

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -0,0 +1,122 @@
+/*
+ * This is a manifest file that'll be compiled into application.css, which will include all the files
+ * listed below.
+ *
```
Comment (Callbritton) [inline app/assets/stylesheets/application.scss:4]: I'm so glad that we were able to get this setup from the beginning! So often styling ends up getting put off and then overlooked in the final push.

File: bin/yarn [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 APP_ROOT = File.expand_path('..', __dir__)
 Dir.chdir(APP_ROOT) do
-  begin
-    exec "yarnpkg", *ARGV
-  rescue Errno::ENOENT
-    $stderr.puts "Yarn executable was not detected in the system."
-    $stderr.puts "Download Yarn at https://yarnpkg.com/en/docs/install"
-    exit 1
-  end
+  exec 'yarnpkg', *ARGV
```
Comment (Callbritton) [inline bin/yarn:6]: Why did we implement Yarn?

**Review by Oxalisviolacea — APPROVED** [review]
This looks great, thanks!

**Comment by Callbritton** [comment]
Looking forward to seeing the new styling in deployment!

File: .gitignore [hunk 3]
```diff
@@ -26,10 +26,15 @@
...
+/config/application.yml
+=======
 <<<<<<< HEAD
 # Ignore application configuration
 /config/application.yml
 =======
 # Ignore test coverage
 /coverage/*
 >>>>>>> 9fbd76410231ff6af04d18bfee68aa8c667b268c
+>>>>>>> 59862269cbcbda23a6902c2920aa5b2edb049aa2
```
Comment (jakeheft) [inline .gitignore:40]: We'll clean all this up (I know it's just gitignore so not a big deal)

File: app/assets/stylesheets/application.scss (hunk 1 above)
Comment (jakeheft) [reply app/assets/stylesheets/application.scss:4]: Yeah site looks great!

File: spec/features/welcome/welcome_spec.rb [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+      app_description = "Moving to a new place is as exciting as it is stressful. Relocate was developed to help alleviate the stress so you can focus on settling in. What makes a home isn’t just the roof you sleep under, but the neighborhood around you. The average person moves approximately 11.7 times in their lifetime, and that means new gyms, new utilities, and new health care providers, among other things. Relocate's user-friendly interface is a quick way to find and compare all the local businesses and necessities around you based on the location you provide."
+      visit '/'
+
+      expect(page).to have_content('Welcome to Relocate')
+      expect(page).to have_link('Login with Google')
+      expect(page).to have_content(app_description)
+      expect(page).to have_link('Privacy Policy')
+    end
+
+    xit "starts the oauth process when you click on the 'Login with Google' link" do
```
Comment (jakeheft) [inline spec/features/welcome/welcome_spec.rb:17]: Remove x to put test back in

**Review by jakeheft — APPROVED** [review]

**Pull request merged**